### ISSUE TITLE
wmp9 on 64 bit prefix not working on Wine 5.7 or newer

### ISSUE THREAD

**Issue by rodrigo21** [Winetricks/winetricks#1562]
Since commit this [change](https://github.com/Winetricks/winetricks/commit/d953037c574fa5eeec3700195a8b7c8ce9c189fd) wmp9 install on 64 bit prefix fails. 

I think it's because winecfg cannot set windows version to be win2k on 64 bit prefix.

**Comment by rmi1974** [comment]
Correct, Windows XP is the minimum version requirement for 64-bit WINEPREFIX. Everything below requires a 32-bit WINEPREFIX via `WINEARCH=win32`

There should be a blocker condition for recipes that need to set WinVer < Window XP to require a 32-bit WINEPREFIX.

**Comment by austin987** [comment]
Actually, wmp9 installer supports winxp, so this is fixable. Others should get blockers though, yes.

**Comment by kozross** [comment]
Curious if this is still an issue, or whether it's been fixed, but no new release has been cut with this fix?

**Comment by austin987** [comment]
Still an issue; I have a fix committed locally, but haven't pushed things yet (testing/rebasing things).

**Comment by austin987** [comment]
I'm still splitting/testing, but if you're able, please test with:
https://raw.githubusercontent.com/austin987/winetricks/wip/src/winetricks

branch is at:
https://github.com/austin987/winetricks/tree/wip

There's still work to be done on the w_package_broken* bits before I push any of it, but it's close to ready, so I'd like others to test before I push anything.
